bundles/EcommerceFrameworkBundle/PricingManager/Rule.php

```diff
@@ -97,7 +97,7 @@ public static function getById($id)
     /**
      * @var bool
      */
-    protected $active;
+    protected bool $active = false;
```

Comment:
since you touched this file, could you please move the `getById` method under properties for better readability? thanks!